js_modules/dagster-ui/packages/ui-core/src/workspace/CodeLocationRowSet.tsx

```diff
@@ -47,9 +49,18 @@ export const ImageName = ({metadata}: {metadata: WorkspaceDisplayMetadataFragmen
       <ImageNameBox flex={{direction: 'row', gap: 4}}>
         <span style={{fontWeight: 500}}>image:</span>
         <Tooltip content="Click to copy" placement="top" display="block">
-          <button onClick={onClick}>
+          <UnstyledButton
+            onClick={onClick}
+            style={{
```

Comment:
Maybe we should name this style? And just accumulate a list of these styles in the UnstyledButton file? Might make it easier to consolidate down the line / figure out if we should standardize any of the styles if they're used often enough.

Reply:
Yeah, sounds reasonable. This one is a little strange because it's got the width and display values set, which might not be super reusable, but I could at least share them internally for the use cases here.